Installation/cmake/modules/CGAL_target_use_OpenGR.cmake

```diff
@@ -0,0 +1,9 @@
+if (CGAL_target_use_OpenGR_included)
+  return()
+endif()
+set(CGAL_target_use_OpenGR_included TRUE)
+
+function(CGAL_target_use_OpenGR target)
+  target_include_directories(${target} PUBLIC ${OpenGR_INCLUDE_DIR})
+  target_compile_options( ${target} PUBLIC -DCGAL_LINKED_WITH_OPENGR)
+endfunction()
```

Comment:
@sgiraudot Why not link with the imported targets defined by `OpenGRConfig.cmake`?

That would solve the requirement for C++17: CGAL targets using OpenGR would be compiled with C++17 enabled (and automatically, using CMake features).

Cc: @nmellado

Reply:
This should be indeed exported, I made changes for that on `master` (https://github.com/STORM-IRIT/OpenGR/commit/ad80cacbdb5cc9bf034e9a9613f1982105e6708d), see https://github.com/STORM-IRIT/OpenGR/blob/7da3b75a6b153ea243f9190fd62a8d189d9ba70f/src/gr/accelerators/CMakeLists.txt#L37-L39.

Note also that the library is exported in a cmake namespace `gr`, so `target_link_library( ... gr::XXX)` should be enough to get your target properly configured (maybe I am missing something here).